api/pkg/apis/v1alpha1/providers/stage/http/http_test.go

```diff
@@ -73,6 +74,35 @@ func TestPingBing(t *testing.T) {
 	assert.Equal(t, 200, outputs["status"])
 	assert.NotNil(t, outputs["body"])
 }
+func TestPostRequestWithJson(t *testing.T) {
+	provider := HttpStageProvider{}
+	err := provider.Init(HttpStageProviderConfig{
+		Method: "POST",
+		Url:    "https://jsonplaceholder.typicode.com/posts",
+	})
```

Comment:
what's this url?

Reply:
It's a POST request testing mock website